### DISCUSSION TITLE
Real-time collaboration

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2267] [Feature Ideas]
Currently, Mathesar does not notify users or reflect changes that other users have made to the data they are currently looking at. We'd like to eventually support real-time collaboration so that changes made elsewhere are immediately reflected in the Mathesar UI.

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by martindemello** [comment]
hey, just wanted to check and see if this is on the concrete roadmap yet . i've talked to some of the mathesar team about my use case for this in the past, but to summarise, i currently have a "database" sitting in a google sheet, one table to a tab, and i would love to replace it with mathesar, but the ability to have multiple people collaborate on it and see each other's edits reflected instantly is a killer feature for google sheets and it outweighs any other clunkiness arising from not using a real db
